content/articles/close-account.md

```diff
@@ -19,16 +19,29 @@ To delete yourself as a user, you'll first need to [unsubscribe your accounts fr
 <div class="section-steps" markdown="1">
 ##### To permanently delete yourself as a user, along with all your domains and records:
 
+1. [Unsubscribe your accounts from your plan(s) at DNSimple](/articles/cancel-subscription).
+
 1. Go to [your User Settings page](https://dnsimple.com/user):
 
-    ![Navigate to user settings page](/files/access-user-settings.jpg)
+    **New screenshot** 
+
+1. Click <label>Delete user</label> at the bottom of the page:
 
-1. Click on the <label>Delete user</label> button at the bottom of the page:
+    **New screenshot**
 
-     ![Delete user](/files/delete-user.png)
+   <info>
+   The <label>Delete user</label> button will only be available if you have no active DNSimple subscriptions.
+   </info>
+   
+1. Include your feedback, or write anything you like in the provided field, then click **I understand the consequences, close this user and account**. The field cannot be left blank.
 
-   The <label>Delete user</label> button will only show up if you have no active DNSimple subscriptions.
+   **New screenshot**
```

Comment:
https://drive.google.com/file/d/1OFQYmEOe8REyTN6yw4W-Y43BzQQopDNN/view?usp=sharing

Reply:
@dperrymorrow These screenshots were not added. Please reopen and add the screenshots.